schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
-CREATE TABLE animals (
-    name varchar(100)
+CREATE TABLE vet_clinic.animals (
+    id INT,
+    name VARCHAR(100),
+    date_of_birth DATE,
+    escape_attempts INT,
+    neutered BOOLEAN,
+    weight_kg FLOAT
```

Comment:
- [ ] @BHObserver Great work adding the animals' table, however `the weigth_kg` column should be of type decimal, type float stores approximate values while type decimal stores exact values, kindly consider changing the datatype.
![image](https://github.com/BHObserver/Vet-Clinic-Database/assets/61408860/280d4e55-6211-4891-8684-473dddda7825)